index.yaml

```diff
@@ -101,6 +101,13 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: FeedbackMessageModel
```

Comment:
Er -- my question still stands, though. What causes this index to be generated? We ought to know that.

If you're not sure about the code bit, what actions on the devserver cause this index to be generated?